playbooks/generic-deploy_airship.yml

```diff
@@ -4,7 +4,7 @@
   any_errors_fatal: true
   roles:
     - role: airship-setup-deployer
-      when: redeploy_osh_only is not defined or redeploy_osh_only != true
+      when: redeploy_osh_only is not defined or not redeploy_osh_only
```

Comment:
if ``redeploy_osh_only`` was not passed through boolean filter, you should do so here. But else it's better.

You can also have ``not (redeploy_osh_only | default(False) | bool)``. Or better, rewrite the  variable to be not double negative.